README.md

```diff
@@ -17,6 +17,9 @@ And an example conversation:
 
 <img src="https://user-images.githubusercontent.com/9945255/116060273-054fb080-a682-11eb-9fe4-d864305bf4d2.png" width="300" >
 
+## Architecture design
```

Comment:
It's better to use a local link `[here](docs/design.md)`

Reply:
OK, never knew this was possible... 🤯  I've changed it.